README.md

```diff
@@ -307,3 +315,7 @@ We already have email plugin: https://github.com/globocom/azkaban-jobtype-email
 ## Contribute
 
 For development and contributing, please follow [Contributing Guide](https://github.com/globocom/auror-core/blob/master/CONTRIBUTING.md) and ALWAYS respect the [Code of Conduct](https://github.com/globocom/auror-core/blob/master/CODE_OF_CONDUCT.md)
+
+## Image credits 
+
+<span>Photo by <a href="https://unsplash.com/@rhii?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Rhii Photography</a> on <a href="https://unsplash.com/?utm_source=unsplash&amp;utm_medium=referral&amp;utm_content=creditCopyText">Unsplash</a></span>
```

Comment:
You can put this right after the image, seems more clear and understandable!

Reply:
Done